openfisca_core/formulas.py

```diff
@@ -434,7 +434,7 @@ def compute(self, period, **parameters):
         except legislations.ParameterNotFound as exc:
             if exc.variable_name is None:
                 raise legislations.ParameterNotFound(
-                    instant = exc.instant,
+                    instant_str = exc.instant_str,
```

Comment:
In this case I think it is justified (though ugly, I agree).

In openfisca, dates (called "instant") are stored unpredictably as `str` or `period.Instant`. I already introduced many bugs (and spent a lot of time debugging them) because of the ambiguity.